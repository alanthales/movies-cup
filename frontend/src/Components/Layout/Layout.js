import React from "react";
import Grid from "@material-ui/core/Grid";
import Header from "../Header/Header";

const Layout = ({ title, subtitle, children }) => (
  <Grid container direction="column" spacing={2}>
    <Grid item>
      <Header title={title} subtitle={subtitle} />
    </Grid>

    <Grid item>
      {children}
    </Grid>
  </Grid>
);

export default Layout;