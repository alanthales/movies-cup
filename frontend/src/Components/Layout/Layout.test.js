import React from "react";
import Layout from "./Layout";
import renderer from "react-test-renderer";

test("render Layout from given props", () => {
  const component = renderer.create(
    <Layout title="Test" subtitle="Testing layout" />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})