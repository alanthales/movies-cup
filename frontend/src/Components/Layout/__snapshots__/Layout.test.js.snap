// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Layout from given props 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiBox-root MuiBox-root-105"
    >
      <div
        className="MuiGrid-root header-component MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
        >
          <h4>
            CAMPEONATO DE FILMES
          </h4>
          <h1>
            Test
          </h1>
          <hr
            style={
              Object {
                "width": "50px",
              }
            }
          />
          <p>
            Testing layout
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  />
</div>
`;
